example-wasm/src/components/TaskItem.tsx

```diff
@@ -0,0 +1,32 @@
+import { Task } from '../models/Task';
+import styles from '../styles/TaskItem.module.css';
+
+type TaskItemProps = {
+  task: Task;
+  onToggleStatus: (task: Task) => void;
+  onDelete: (task: Task) => void;
+};
+
+export function TaskItem({ task, onToggleStatus, onDelete }: TaskItemProps) {
```

Comment:
Note: Removed memoization of this component due to `prevProp.task.isComplete` always being equal to `nextProp.task.isComplete` despite toggling it via the UI. See the [remove commit](https://github.com/realm/realm-js/pull/5880/commits/3c76de9409ec5ec67f16f740ef1650f95b6cde2a) for more context.